modules/hal_nordic/nrfx/CMakeLists.txt

```diff
@@ -197,6 +197,7 @@ endif()
 zephyr_compile_definitions_ifdef(CONFIG_SOC_NRF54LX_SKIP_CLOCK_CONFIG NRF_SKIP_CLOCK_CONFIGURATION)
 zephyr_compile_definitions_ifdef(CONFIG_SOC_NRF54LX_DISABLE_FICR_TRIMCNF NRF_DISABLE_FICR_TRIMCNF)
 zephyr_compile_definitions_ifdef(CONFIG_SOC_NRF54LX_SKIP_GLITCHDETECTOR_DISABLE NRF_SKIP_GLITCHDETECTOR_DISABLE)
+zephyr_compile_definitions_ifndef(SOC_NRF54L_ERRATA_56_WORKAROUND NRF54L_CONFIGURATION_56_ENABLE=0)
```

Comment:
Isn't CONFIG_ prefix missing there?

Reply:
Yes, it is - @ankuns already noticed this in the upstream. I have now updated the PR. Thanks!